app/Http/Controllers/UserController.php

```diff
@@ -59,15 +59,30 @@ public function show(User $user)
      */
     public function edit(User $user)
     {
-        //
+        return view('auth.profile',
+            [
+                'heading' => 'Welkom '. Auth::user()->name,
+                'user' => $user
+            ]
+        );
     }
 
     /**
      * Update the specified resource in storage.
      */
-    public function update(Request $request, User $user)
+    public function update(Request $request, $id)
```

Comment:
Waarom hier naar id veranderd en bij de edit method niet. Ik denk dat we beter consistent kunne blijven met of we dit wel of niet doen.